rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -49,9 +49,9 @@ include::modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc[le
 //include::modules/rosa-sts-associating-your-aws-account.adoc[leveloffset=+2]
 include::modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc[leveloffset=+1]
 [id="rosa-hcp-creating-vpc"]
-= Creating a Virtual Private Cloud for your ROSA {hcp} clusters
+== Creating a Virtual Private Cloud for your {hcp-title} clusters
 
-You must have a Virtual Private Cloud (VPC) to create {hcp} for ROSA cluster. You can manually create these using the AWS console, or you can use `terraform` to create a VPC by using a template.
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You can manually create these using the AWS console, or you can use `terraform` to create a VPC by using a template.
```

Comment:
Would this read better if it was "You must have a a Virtual Private Cloud (VPC) to create *a* ROSA with hosted control planes cluster." Or just pluralize clusters.